frontend/server.js

```diff
@@ -57,14 +57,14 @@ const runSessionDbMigration = async (pool) => {
 };
 
 async function main() {
-  const dev = process.env.NODE_ENV !== "production";
-  const dotEnvFileSegment = dev ? "development" : "production";
+  let env = process.env.NODE_ENV || "development";
```

Comment:
I think that can be `const`

```suggestion
  const env = process.env.NODE_ENV || "development";
```

Reply:
Ha! yeah, left over from refactor. Thanks